doc/source/api.rst

```diff
@@ -6,6 +6,19 @@ can use to generate, consume, modify and verify in-toto metadata, as a more
 feature-rich, programmable alternative to the :doc:`command line tools
 <command-line-tools/index>`.
 
+.. note::
+
+   **Cryptographic Signatures**
+
+   in-toto metadata is signed with cryptographic keys using the in-house crypto
```

Comment:
nit: maybe we should drop the term "in-house" given in-toto is at the CNCF and securesystemslib is not, I'm not sure how different people may parse it.

Reply:
Fixed in  9ce441a